x/nft/client/testutil/cli_test.go

```diff
@@ -1,15 +1,22 @@
 package testutil
 
 import (
+	_ "embed"
 	"testing"
 
+	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
+	"cosmossdk.io/core/appconfig"
 	"github.com/cosmos/cosmos-sdk/testutil/network"
 )
 
+//go:embed app.yaml
+var appConfig []byte
+
 func TestIntegrationTestSuite(t *testing.T) {
-	cfg := network.DefaultConfig()
+	cfg, err := network.DefaultConfigWithAppConfig(appconfig.LoadYAML(appConfig))
```

Comment:
We should put the app config after it's been loaded in a var that can be used by other tests in this module that need app config. Probably in an internal testutil package

Reply:
👍🏾 added in `x/nft/testutil`